src/app/components/confirm-dialog/confirm-dialog.component.scss

```diff
@@ -0,0 +1,82 @@
+.dialog-container {
+  display: flex;
```

Comment:
put thos group in mixin : 
display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;

its in use a lot